pack.Recycle api changed in 0.10

when I build this plugin,I got this error
```
heka/src/github.com/liuyangc3/heka-redis/input.go:59: not enough arguments in call to pack.Recycle
make[2]: *** [CMakeFiles/hekad] Error 2
make[1]: *** [CMakeFiles/hekad.dir/all] Error 2
```
It seems like the api changed in the leatest version of heka.
http://hekad.readthedocs.org/en/latest/developing/plugin.html#transitional-filter-output-apis-for-v0-10-only
>One important detail about this pattern, however: if for any reason your plugin should pull a PipelinePack off of the input channel and not end up passing it on to one of the Deliver methods, you must call pack.Recycle(nil) to free the pack up to be used again. Failure to do so will eventually deplete the pool of PipelinePacks and will cause Heka to freeze.

--- a/input.go
+++ b/input.go
@@ -80,7 +80,7 @@ func (rpsi *RedisPubSubInput) Run(ir pipeline.InputRunner, h pipeline.PluginHelp
 				if e != nil {
 					ir.LogError(fmt.Errorf("Couldn't parse Redis message: %s", n.Data))
 				}
-				pack.Recycle()
+				pack.Recycle(nil)
 			}
 		case redis.Subscription:
 			ir.LogMessage(fmt.Sprintf("Subscription: %s %s %d\n", n.Kind, n.Channel, n.Count))

